src/main/java/uk/gov/ons/fwmt/job_service/service/impl/JobServiceImpl.java

```diff
@@ -49,9 +48,10 @@ public JobServiceImpl(
   @Override
   public SampleSummaryDTO processSampleFile(MultipartFile file)
           throws IOException, InvalidFileNameException, MediaTypeNotSupportedException{
-    jobResourceService.sendCSV(file);
     File f = convertFile(file);
     SampleSummaryDTO sampleSummaryDTO = validateSampleFile(f);
+    boolean valid = sampleSummaryDTO.getUnprocessedRows().isEmpty();
```

Comment:
directly assign the value in arg rather than assign to a variable not used anywhere else.
for e.g.: jobResourceService.sendCSV(f,sampleSummaryDTO.getUnprocessedRows().isEmpty())